pyconcz/static_src/scss/pyconcz19/_home.scss

```diff
@@ -12,9 +12,9 @@
 .pc-main-lead {
     position: relative;
     background-color: $black;
-    background-image: url(/2019/static/img/ostrava/20150413_zer_hlubina_044.jpg);
+    background-image: url(/2019/static/img/ostrava/vitkovice.jpg);
     background-repeat: no-repeat;
-    background-position: 50% 25%;
+    background-position: 50% 75%;
```

Comment:
This needs a rebuild, not sure how to do it.

Reply:
Description in README doesn't help?

Reply:
I'm not doing `sudo npm install gulp-cli -g` ever. Sorry.

Reply:
But I haven't read the README first, sorry about that. Now I know how to do it, but I don't want to :laughing:  Will try to search for gulp in the RPM repos.

Reply:
`npm install gulp-cli` installs locally, and hides the binary in `./node_modules/.bin/gulp`.

Reply:
I manage to do that in [Docker container](https://hub.docker.com/r/akoehnlein/surf). You just need to start a container with shared folder via `-v` and then run `npm install` and `npm run build`.

Do you want to try it or should I send a PR on your fork?

Reply:
feel free to push directly to my fork.

Reply:
I will update npm (frontend) stack ASAP to remove any global installation.

Reply:
#28